pyproject.toml

```diff
@@ -103,15 +103,52 @@ before-all = ".github/workflows/wheels-dependencies.sh"
 build-verbosity = 1
 
 config-settings = "raqm=enable raqm=vendor fribidi=vendor imagequant=disable"
-# Disable platform guessing on macOS
-macos.config-settings = "raqm=enable raqm=vendor fribidi=vendor imagequant=disable platform-guessing=disable"
 
 test-command = "cd {project} && .github/workflows/wheels-test.sh"
 test-extras = "tests"
+test-requires = [
+  "numpy",
+]
+xbuild-tools = [  ]
+
+[tool.cibuildwheel.macos]
+# Disable platform guessing on macOS to avoid picking up Homebrew etc.
+config-settings = "raqm=enable raqm=vendor fribidi=vendor imagequant=disable platform-guessing=disable"
 
 [tool.cibuildwheel.macos.environment]
+# Isolate macOS build environment from Homebrew etc.
 PATH = "$(pwd)/build/deps/darwin/bin:$(dirname $(which python3)):/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin"
 
+[tool.cibuildwheel.ios]
+# Disable platform guessing on iOS, and disable raqm (since there won't be a
+# vendor version, and we can't distribute it due to licensing)
+config-settings = "raqm=disable imagequant=disable platform-guessing=disable"
+
+# iOS needs to be given a specific pytest invocation and list of test sources.
+test-sources = [
+  "checks",
+  "Tests",
+  "pyproject.toml",
+  "selftest.py",
+]
+test-command = [
+  "python -m selftest",
+  "python -m pytest -vv checks/check_wheel.py Tests",
```

Comment:
```suggestion
  "python -m pytest -v checks/check_wheel.py Tests",
```

Reply:
In case of success, there's no real difference; but in case of failure, `-vv` will sometimes include failure context (such as complete diffs) that is truncated by `-v`. If we *can* put the full context in the logs, why not do so?

Reply:
Ok then. I've created https://github.com/python-pillow/Pillow/pull/9040 to suggest this change elsewhere.